src/mgr/PyModule.h

```diff
@@ -74,6 +74,8 @@ class PyModule
 
   // Populated if loaded, can_run or failed indicates a problem
   std::string error_string;
+  // relative path from the mgr_module_path where python dependencies might be embeded
+  const std::string pypackages_path;
```

Comment:
The goal of using a const was to make it stand out (top of the file, stand-alone declaration + assignment, etc), but in this way the const literal is still buried within the code, so what about?:

From [here](https://stackoverflow.com/a/1563906):
```suggestion
  inline static const std::string pypackages_path = "__pypackages__/lib/python3/site-packages";
```
Or [this one](https://stackoverflow.com/a/43335753):
```suggestion
  constexpr std::string_view pypackages_path = "__pypackages__/lib/python3/site-packages";
```

BTW, not sure if Ceph uses a specific type for path strings, but since C++17 there's [this one](https://en.cppreference.com/w/cpp/filesystem/path).